src/CareTogether.Core/LogType.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+namespace CareTogether
+{
+    public enum LogType
+    {
+        COMMUNITY_EVENT, CONTACT_COMMAND_EXECUTED_EVENT, GOAL_COMMAND_EXECUTED_EVENT, REFERRAL_EVENT
```

Comment:
Use regular CamelCasing for these names, per C# convention:
```suggestion
        CommunityEvent, ContactCommandExecutedEvent, GoalCommandExecutedEvent, ReferralEvent
```

Reply:
Also, you can change these names to make the generated filenames look friendlier in Blob storage, similar to what you were doing with the dictionary in _Extensions.cs_. Maybe try:
```suggestion
        CommunityEventLog, ContactsEventLog, GoalsEventLog, ReferralsEventLog
```

Reply:
My Java background rearing its head here :)